torch/testing/__init__.py

```diff
@@ -297,6 +297,9 @@ def __add__(self, other):
 def floating_types():
     return _floating_types
 
+def floating64_type_and(*dtypes):
```

Comment:
Per the note above on line 282, these functions are intended to correspond 1:1 with the C++ dispatch macros, so let's not add a new function here.